gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -120,7 +122,7 @@ public class GobblinServiceJobScheduler extends JobScheduler implements SpecCata
   protected final Optional<UserQuotaManager> quotaManager;
   protected final Optional<FlowTriggerHandler> flowTriggerHandler;
   @Getter
-  protected final Map<String, Spec> scheduledFlowSpecs;
+  protected final Map<String, FlowSpec> scheduledFlowSpecs;
```

Comment:
NBD, this may be for the best... but I can't seem to figure out: where is the initial motivation coming in?  e.g. I was looking for a changed method signature, but didn't notice one.  is it entirely a preference to change the type of these various members?  (again, I'm not calling into question the decision to change, more wanting to understand motivation for the decision.)

Reply:
I believe the `initial commit` was too big and some imperfection was missed out.

Reply:
aah, ok, I see - so essentially just following more widely the specific-derived-class (`FlowSpec`) typing instituted therein